docs/CHANGELOG.asciidoc

```diff
@@ -70,6 +71,98 @@ Review important information about the {kib} 7.16.x releases.
+{kibana-ref-all}/7.12/release-notes-7.12.0.html#breaking-changes-7.12.0[7.12] | {kibana-ref-all}/7.11/breaking-changes-7.11.html[7.11] | 
+{kibana-ref-all}/7.10/breaking-changes-7.10.html[7.10] | {kibana-ref-all}/7.9/breaking-changes-7.9.html[7.9] | {kibana-ref-all}/7.8/breaking-changes-7.8.html[7.8] | 
+{kibana-ref-all}/7.7/breaking-changes-7.7.html[7.7] | {kibana-ref-all}/7.6/breaking-changes-7.6.html[7.6] | {kibana-ref-all}/7.5/breaking-changes-7.5.html[7.5] |
+{kibana-ref-all}/7.4/breaking-changes-7.4.html[7.4] | {kibana-ref-all}/7.3/breaking-changes-7.3.html[7.3] | {kibana-ref-all}/7.2/breaking-changes-7.2.html[7.2] | 
+{kibana-ref-all}/7.1/breaking-changes-7.1.html[7.1] | {kibana-ref-all}/7.0/breaking-changes-7.0.html[7.0]
+
+[float]
+[[enhancements-and-bug-fixes-v7.17.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 7.17.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v7.17.0]]
+==== Enhancements
+Elastic Security::
+For the Elastic Security 7.17.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Lens & Visualizations::
+* Allow users to not render suggestions in *Lens* {kibana-pull}115946[#115946]
+* Avoid to compute over time suggestion for expensive configurations in *Lens* {kibana-pull}115932[#115932]
+Monitoring::
+Adds telemetry stats for legacy notifications system {kibana-pull}122472[#122472]
+Operations::
+Replaces CentOS base image with Ubuntu {kibana-pull}118928[#118928]
+Platform::
+* Response logs emitted by the elasticsearch service include the size of the response body {kibana-pull}123173[#123173]
+* Adds event loop delay histogram percentiles to `metrics.ops` logger message and meta field {kibana-pull}120451[#120451]
+
+[float]
+[[fixes-v7.17.0]]
+==== Bug Fixes
+Alerting::
+Fixes call to update index alias to hidden {kibana-pull}122882[#122882]
+Canvas::
+* Fixes an issue where a workpad element was breaking after changing the expression from expression editor and resizing {kibana-pull}122154[#122154]
+* Repeat image bug with image height fixed {kibana-pull}121497[#121497]
+Discover::
+* Fixes saved search hidden chart can't be opened when returning to Discover {kibana-pull}122745[#122745]
+* Fixes page up and down navigation in classic table {kibana-pull}122742[#122742]
+* Fixes context view for document ids containing special characters {kibana-pull}122737[#122737]
+* Adds {kib} services provider for embeddable {kibana-pull}121621[#121621]
+Elastic Security::
+For the Elastic Security 7.17.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Lens & Visualizations::
+* Restores default chart `timezone` to `'local'` in XY charts {kibana-pull}123264[#123264]
```

Comment:
This did not affect Lens and Visualizations (I think it fixed a bug in Observability)